packages/babel-generator/src/printer.ts

```diff
@@ -598,11 +598,6 @@ class Printer {
     if (this._printedComments.has(comment)) return;
     this._printedComments.add(comment);
 
-    if (comment.start != null) {
-      if (this._printedCommentStarts[comment.start]) return;
```

Comment:
It doesn't look  like `this._printedCommentStarts` is referenced anywhere else in this class, so we should be able to remove that class property.